Reading List/Reading List/Table Views/BookTableViewCell.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  BookTableViewCell.swift
+//  Reading List
+//
+//  Created by Richard Gibbs on 5/19/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import UIKit
+
+class BookTableViewCell: UITableViewCell, BookTableViewCellDelegate {
+    func toggleHasBeenRead(for cell: BookTableViewCell) {
```

Comment:
The delegate function does not go here. Where would this be most useful or which file needs to do something with this function?